src/main/java/io/pivotal/literx/Part06Request.java

```diff
@@ -17,14 +17,14 @@
 
 //========================================================================================
 
-	// TODO Create a StepVerifier that initially requests all values and expect 4 values to be received
+	// TODO Create a StepVerifier that expects 4 values to be received and requests all remaining values
 	StepVerifier requestAllExpectFour(Flux<User> flux) {
 		return null;
 	}
 
 //========================================================================================
 
-	// TODO Create a StepVerifier that initially requests 1 value and expects User.SKYLER then requests another value and expects User.JESSE.
+	// TODO Create a StepVerifier that initially requests 1 value and expects User.SKYLER then requests another value and expects User.JESSE then stops request.
```

Comment:
could be made even more explicit by stating "then stops verifying by cancelling the source"